File: 30-days-of-code/Day08/Maps.java
/* ************************************************************************** */
/*                                                                            */
/*                                                        :::      ::::::::   */
/*   Maps.java                                          :+:      :+:    :+:   */
/*                                                    +:+ +:+         +:+     */
/*   By: alaamimi <alaamimi@student.42.fr>          +#+  +:+       +#+        */
/*                                                +#+#+#+#+#+   +#+           */
/*   Created: 2021/02/26 16:45:01 by alaamimi          #+#    #+#             */
/*   Updated: 2021/02/26 16:45:04 by alaamimi         ###   ########.fr       */
/*                                                                            */
/* ************************************************************************** */

import java.util.*;
import java.io.*;

public class Solution
{
	public static void main(String[] args)
	{
		Scanner scanInput = new Scanner(System.in);
		int n = scanInput.nextInt();

		Map<String,Integer> phoneBook = new HashMap<>();
		for(int i = 0; i < n; i++)
		{
			String name = scanInput.next();
			int phone = scanInput.nextInt();
			phoneBook.put(name, phone);
		}
		while(scanInput.hasNext())
		{
			String str = scanInput.next();
			if (phoneBook.containsKey(str))
				System.out.println(str + "=" + phoneBook.get(str));
			else
				System.out.println("Not found");
		}
		scanInput.close();
	}
}

File: 30-days-of-code/Day08/maps.py
# **************************************************************************** #
#                                                                              #
#                                                         :::      ::::::::    #
#    maps.py                                            :+:      :+:    :+:    #
#                                                     +:+ +:+         +:+      #
#    By: alaamimi <alaamimi@student.42.fr>          +#+  +:+       +#+         #
#                                                 +#+#+#+#+#+   +#+            #
#    Created: 2021/03/07 14:24:27 by alaamimi          #+#    #+#              #
#    Updated: 2021/03/07 14:24:47 by alaamimi         ###   ########.fr        #
#                                                                              #
# **************************************************************************** #

import sys

n = int(sys.stdin.readline().strip())
phoneBook = dict()

for i in range(0, n):
    entry = sys.stdin.readline().strip().split(' ')
    phoneBook[entry[0]] = entry[1]

query = sys.stdin.readline().strip()
while query:
    phoneNumber = phoneBook.get(query)
    if phoneNumber:
        print(query + '=' + phoneNumber)
    else:
        print('Not found')
    query = sys.stdin.readline().strip()